tests/packagedcode/test_plugin_package.py

```diff
@@ -127,6 +127,15 @@ def test_package_command_scan_phpcomposer(self):
         run_scan_click(['--package', '--strip-root', '--processes', '-1', test_dir, '--json', result_file])
         check_json_scan(expected_file, result_file, remove_uuid=True, regen=REGEN_TEST_FIXTURES)
 
+    # just for test purpose, used change-case example
+    def test_plugin_package_with_package_summary(self):
+        test_dir = self.get_test_loc('package_summary/change-case-change-case-5.4.4.zip-extract') 
+        result_file = self.get_temp_file('json')
+        expected_file = self.get_test_loc('package_summary/expected.json')
+
+        run_scan_click(['--package', '--strip-root', '--processes', '-2','--package-summary','--json-pp', result_file, test_dir])
```

Comment:
```suggestion
        run_scan_click(['--package', '--strip-root', '--processes', '-1','--package-summary','--json-pp', result_file, test_dir])
```
This is disabling multiprocessing. See the scan options on the test above.

Reply:
Resolved.